doc/devel/MEP/MEP12.rst

```diff
@@ -192,10 +192,10 @@ the entry point to these examples, and sections could really help users
 navigate the gallery. Thus, tags are complementary to this reorganization.
 
 
-.. _PEP8: http://www.python.org/dev/peps/pep-0008/
+.. _PEP8: https://www.python.org/dev/peps/pep-0008/
 
-.. [1] http://github.com/matplotlib/matplotlib/pull/714
-.. [2] http://github.com/matplotlib/matplotlib/issues/524
+.. [1] https://github.com/matplotlib/matplotlib/pull/714
+.. [2] https://github.com/matplotlib/matplotlib/issues/524
 .. [3] http://matplotlib.1069221.n5.nabble.com/Matplotlib-gallery-td762.html#a33379091
-.. [4] http://www.loria.fr/~rougier/teaching/matplotlib/
-.. [5] http://www.loria.fr/~rougier/coding/gallery/
+.. [4] http://www.labri.fr/perso/nrougier/teaching/matplotlib/
```

Comment:
It might be a typo on that website, yes!

/perso/ works, but http://www.labri.fr/person/nrougier/teaching/matplotlib/ gives a 404